dev/com.ibm.ws.security.spnego.fat.common/fat/src/com/ibm/ws/security/spnego/fat/config/Krb5Helper.java

```diff
@@ -114,6 +118,78 @@ public Subject kerberosLogin(LibertyServer server, String userName, String passw
         return subject;
     }
 
+    //ToDo: add if we ever need to run without jaas.conf
```

Comment:
We can remove it, however, if we do, when I perform the squash it will no longer be in git history.

Leaving it for now in case we do need to use it in the future, will help make it quickly able to be enabled by simply un-commenting.

Reply:
Do you think there is a chance we will use this in the near future? Not sure if having this will cause confusion.

Reply:
I think there is some chance. I expect I will remember that issue for quite some time and why I put the commented part there. 

Since we have discussed it here on the PR, I could remove it from the code, and just come back to this PR to reference it if needed in the future

Reply:
sounds good